wrapanapi/systems/openstack.py

```diff
@@ -397,6 +397,10 @@ def __init__(self, system, raw=None, **kwargs):
     def _identifying_attrs(self):
         return {'uuid': self._uuid}
 
+    @property
+    def created_at(self):
```

Comment:
Please be consistent with existing implementations of these properties, as our goal here is to have an API we can apply across all of the supported providers and their entities.

https://github.com/ManageIQ/wrapanapi/blob/master/wrapanapi/systems/rhevm.py#L62

https://github.com/ManageIQ/wrapanapi/blob/master/wrapanapi/systems/virtualcenter.py#L644

https://github.com/ManageIQ/wrapanapi/blob/master/wrapanapi/systems/ec2.py#L109

Additionally, if this attribute is available on the raw object for instances and images, consider enhancing this module by creating a SharedMethodsMixin class, inherited by `OpenstackImage` and `OpenstackInstance`, providing this attribute for both types.

Reply:
Done